experimental/packages/opentelemetry-instrumentation/src/semconvStability.ts

```diff
@@ -54,17 +69,10 @@
  *  }
  *
  */
-
-export const enum SemconvStability {
-  /** Emit only stable semantic conventions. */
-  STABLE = 0x1,
-  /** Emit only old semantic conventions. */
-  OLD = 0x2,
-  /** Emit both stable and old semantic conventions. */
-  DUPLICATE = 0x1 | 0x2,
-}
-
-function _semconvStabilityFromStr(marker: string, str: string | undefined) {
+export function semconvStabilityFromStr(
+  namespace: string,
```

Comment:
Suggestion:

This would give intellisense suggestions while also allowing arbitrary strings for those users who need them (maybe an area we haven't added to the list yet).

```suggestion
  namespace: SemconvStabilityNamespace,
```

```typescript
type SemConvStabilityNamespace = "http" | "messaging" | "database" | (string & {});
```

<img width="820" alt="image" src="https://github.com/user-attachments/assets/f2e721a9-5e0d-4cfa-8041-22d9b877af95" />

Reply:
Sorry for the delay. Added in commit 64110b42b. I also added the "k8s" namespace because TIL: https://opentelemetry.io/docs/specs/semconv/non-normative/k8s-migration/